### ISSUE TITLE
ZeroMQ libarary on macOS

### ISSUE THREAD

**Issue by Francisco-DAnconia** [cculianu/Fulcrum#134]
How can I know if ZeroMQ communication is functioning with Fulcrum?

I build Fulcrum in a virtual machine and then move it to my node machine. Within the VM, I installed MacPorts, then built zmq into /opt/local. (FWIW I did have to add the qmake flags) 

MacPorts installed zmq 4.3.4 (libzmq.5.2.4.dylib)

The build output included: 
- Project MESSAGE: CLI overrides: LIBS=-L/opt/local/lib -lzmq
- Project MESSAGE: CLI overrides: INCLUDEPATH=/opt/local/include
- Project MESSAGE: ZMQ: using CLI override
- ld: warning: dylib (/opt/local/lib/libzmq.dylib) was built for newer macOS version (12.0) than being linked (10.14)

On the node machine, zmq 4.3.4 was installed via brew in /usr/local/…

 When I launch Fulcrum I see:
- zmq: libzmq version: 4.3.4, cppzmq version: 4.7.1
- <Controller> Starting ZMQ Notifier (hashblock) ...

Is Fulcrum successfully using ZeroMQ communication? Or is there something else to look for/check?

Thanks

**Comment by cculianu** [comment]
Yes that warning is a BS message from Qt library.  It's safe to ignore and yes it should be working if running the binary reveals it uses that version,.
